Automattic/sensei
# Add progress preview control for modules

### Changes proposed in this Pull Request

* A new control has been added to the module block.
![Sep-16-2020 10-23-18](https://user-images.githubusercontent.com/53191348/93305146-ca8dbc00-f806-11ea-9a96-1f86844b8d02.gif)

### Testing instructions

* Try creating a new course and add a module block.
* While editing the module block, observe the new setting in the sidebard.
* Modify the setting and observe that the module status is changed.
* Observe the module status on the frontend. For now the frontend uses mock data as we will need to refactor that part when we connect to the actual data.
